src/commonTest/kotlin/test/assertk/assertions/StringTest.kt

```diff
@@ -102,6 +102,35 @@ class StringTest {
     }
     //endregion
 
+    //region containsSubstrings
+    @Test fun containsSubstrings_value_contains_passes() {
+        assertThat("test").containsSubstrings("es", "st")
+    }
+
+    @Test fun containsSubstrings_list_contains_passes() {
+        assertThat("test").containsSubstrings(listOf("es", "st"))
+    }
+
+    @Test fun containsSubstrings_value_not_contains_fails() {
```

Comment:
Could you add/replace these with ones that actually take multiple values?

Reply:
good point. even more necessary, now as the method is overloaded.